src/components/ATATTextArea.vue

```diff
@@ -54,7 +54,7 @@
         :counter="maxChars"
       >
       </v-textarea>
-      <ATATErrorValidation 
+      <ATATErrorValidation v-if="showErrorMessages"
```

Comment:
This approach only hides the message, but the validation is still triggered rendering the border red and bold in its error state. Need that not to happen as well.
![Screen Shot 2023-04-27 at 9 39 13 AM](https://user-images.githubusercontent.com/90333349/234880265-ac5308e0-0f16-4148-ae0e-df9098a8a088.png)